### ISSUE TITLE
1.7.1-beta.31 causes Uncaught TypeError: module.watch is not a function

### ISSUE THREAD

**Issue by arggh** [meteor/meteor#10148]
I was trying out the newest Meteor beta release and toying around with [meteor-svelte](https://github.com/meteor-svelte) by @klaussner which seemed to work nicely with Meteor `1.7.0.4`, but after upgrading to `1.7.1-beta.31` I got the error below in the client's console. The app was built without errors.

```
Uncaught TypeError: module.watch is not a function
    at App.html (app.js?hash=3f7c256e4f3ce1f987c65bacfe39918a1d4dce8d:14)
    at fileEvaluate (modules-runtime.js?hash=7f6bef46e4c39734b4c025bc1a8a760520295f53:332)
    at Module.require (modules-runtime.js?hash=7f6bef46e4c39734b4c025bc1a8a760520295f53:234)
    at Module.moduleLink [as link] (modules.js?hash=3f5faed87f3f76d969c5067eea35b898ac43f8df:272)
    at main.js (main.js:1)
    at fileEvaluate (modules-runtime.js?hash=7f6bef46e4c39734b4c025bc1a8a760520295f53:332)
    at Module.require (modules-runtime.js?hash=7f6bef46e4c39734b4c025bc1a8a760520295f53:234)
    at require (modules-runtime.js?hash=7f6bef46e4c39734b4c025bc1a8a760520295f53:254)
    at app.js?hash=3f7c256e4f3ce1f987c65bacfe39918a1d4dce8d:662
```

### Steps to reproduce

1. Clone https://github.com/meteor-svelte/ssr-example
2. Run `meteor  update --release 1.7.1-beta.31`
3. Run `npm install`
4. Run `npm install --save @babel/runtime@latest`
5. Run `meteor` and open your browser's console on `localhost:3000`

**Comment by benjamn** [comment]
This should be fixed if you run `meteor update --release 1.7.1-beta.32`. Please feel free to reopen if you're still having problems.

**Comment by arggh** [comment]
Confirm as fixed, thanks!
